### PR TITLE
Update NAI metadata yaml to incorporate fnf feedbacks

### PR DESCRIPTION
**Pull request by johnliu25**
Update NAI metadata yaml to incorporate fnf feedbacks

### CODE REVIEW

**Review by takirala — APPROVED** [review]

File: services/nutanix-ai/metadata.yaml [hunk 1]
```diff
@@ -54,10 +54,10 @@ overview: |-
       GPT-in-a-Box supports seamless integration with existing Nutanix products, thus providing a robust
       and compliant solution.
 
-  ## Documentation Link
+  ## More Information
   - [Nutanix AI Documentation](https://www.nutanix.com/solutions/ai)
```
Comment (deepakm-ntnx) [inline services/nutanix-ai/metadata.yaml:58]: we wanted to add portal link here rite?

**Review by deepakm-ntnx — APPROVED** [review]
pls see my comment

File: services/nutanix-ai/metadata.yaml (hunk 1 above)
Comment (johnliu25) [reply services/nutanix-ai/metadata.yaml:58]: Could you please elaborate and share which portal link here?

Comment (johnliu25) [reply services/nutanix-ai/metadata.yaml:58]: I will add this in another PR.

**Pull request merged**